### PR TITLE
9879 breakeven input

### PR DESCRIPTION
**Pull request by emitche**
- Adjust contextual information in breakeven table
- Make clear which items are editable and which are calculated and not directly editable
- Adjust columns and headings in breakeven products table (Redmine 8721)
- Match form styling inside editable tables to remaining UI for tables
- Refactor breakeven editable table JS to not use CSS classes to identify elements
- Additional styling inside editable breakeven tables
- Fixes bug (Redmine 10314) where new rows in the breakeven products table were not editable

### CODE REVIEW

File: app/views/admin/loans/questionnaires/_tooltip_calculated_field.html.slim [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.tooltip-container
+  i.fa.ms-tooltip.fa-lock data-message="Auto-calculated field."
```
Comment (emitche) [inline app/views/admin/loans/questionnaires/_tooltip_calculated_field.html.slim]: Use translations.

File: app/views/admin/loans/questionnaires/_tooltip_editable_field.html.slim [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.tooltip-container
+  i.fa.ms-tooltip.fa-edit data-message="Editable field. Click inside the item to edit."
```
Comment (emitche) [inline app/views/admin/loans/questionnaires/_tooltip_editable_field.html.slim]: Use translations.

**Review by meltheadorable — APPROVED** [review]
this looks great

**Comment by emitche** [comment]
Try to address easier-to-address hound errors.

**Pull request merged**